### PR TITLE
RELEASE 09.09

### PR DESCRIPTION
**Pull request by michaltangri**

### CODE REVIEW

**Comment by michaltangri** [comment]
That's a big one

**Review by sebastianczajkowski — APPROVED** [review]
A working state of devbranch, looks good

File: sql/2021-08-30-add-test-user.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+
+insert into User(username, password, isAdmin)
+values
+('username@kainos.com', '$2a$04$cKdU1pCkScsJrT9lCdD/A.44crtlEvwufG5tO/oFEBviwpEwzrNy.', 0), --password
+('admin@kainos.com', '$2a$04$qcm3tUsOfYabxN0Dq61JuOYUEFPhFzYw/dqqVYdXlOxCM91I3vcj2', 1); --admin_password
```
Comment (TheoHodges) [inline sql/2021-08-30-add-test-user.sql:6]: Is it a good idea to have this committed?

File: src/main/java/com/kainos/ea/backend/services/JobRoleService.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    CapabilityService capabilityService;
+
+    @Autowired
+    public JobRoleService(JobRoleRepository jobRoleRepository, BandService bandService, CapabilityService capabilityService) {
+        this.jobRoleRepository = jobRoleRepository;
+        this.bandService = bandService;
+        this.capabilityService = capabilityService;
+    }
+
+    // change to list as its simpler
```
Comment (TheoHodges) [inline src/main/java/com/kainos/ea/backend/services/JobRoleService.java:30]: Comment needs removing

File: src/test/java/com/kainos/ea/backend/services/BandCompetencyServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class BandCompetencyServiceTest {
+
+    @Mock
+    private BandCompetencyRepository bandCompetencyRepository;
+
+    @Test
+    public void when_querying_getCompetenciesByBand_expect_ServiceCalledPassback(){
```
Comment (TheoHodges) [inline src/test/java/com/kainos/ea/backend/services/BandCompetencyServiceTest.java:22]: Inconsistent naming of test

**Review by TheoHodges — CHANGES_REQUESTED** [review]

File: sql/2021-08-30-add-test-user.sql (hunk 1 above)
Comment (mactrz) [reply sql/2021-08-30-add-test-user.sql:6]: I mean the passwords are hashed so it's secure?

Comment (TheoHodges) [reply sql/2021-08-30-add-test-user.sql:6]: Just wanted to check

**Review by TheoHodges — APPROVED** [review]

**Pull request merged**